osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/endpoints/SecretManagementEndpoint.java

```diff
@@ -59,90 +72,143 @@ public SecretManagementEndpoint(final SecretManagementService secretManagementSe
         this.converter = converter;
     }
 
-    @PayloadRoot(namespace = NAMESPACE_URI, localPart = "getSecretsRequest")
-    @ResponsePayload
-    public GetSecretsResponse getSecretsRequest(@RequestPayload final GetSecretsRequest request) throws OsgpException {
-        log.info("Handling incoming SOAP request 'getSecretsRequest' for device {}", request.getDeviceId());
+    private String getCorrelationUidFromHeader(final SoapHeaderElement header) {
```

Comment:
Renamed to `getText`

Reply:
Renamed back to getCorrelationUidFromHeader() because  now in this method the TechnicalException("Missing input: correlation UID") is thrown making it specific for CorrelationUid